pkgs/applications/office/fava/default.nix

```diff
@@ -5,18 +5,18 @@ let
 in
 buildPythonApplication rec {
   pname = "fava";
-  version = "1.7";
+  version = "1.9";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c4eba4203bddaa7bc9d54971d2afeeebab0bc80ce89be1375a41a07c4e82b62f";
+    sha256 = "115r99l6xfliafgkpcf0mndqrvijix5mflg2i56s7xwqr3ch8z9k";
   };
 
   doCheck = false;
 
   propagatedBuildInputs = with python3.pkgs;
```

Comment:
I'm not seeing all of these declared https://github.com/beancount/fava/blob/088d6ba091d718fc42b21051fa35894fbfb22d23/setup.py#L33

Reply:
Thanks for your review! I've added the missing dependencies.